internal/git/util_test.go

```diff
@@ -428,8 +428,8 @@ AAAECl1AymWUHNdRiOu2r2dg97arF3S32bE5zcPTqynwyw50HAtto0bVGTAUATJhiDTjKa
 			Optional:        nil,
 		}},
 	}
-	repoUrL := "ssh://root@localhost:2222/var/www/git/test.git"
-	cloneOptions, err := gitshared.GetRepoCloneOptions(context.Background(), credential, client, repoUrL)
+	repoUrL := "ssh://root@localhost:2222/var/www/git/repo1.git"
+	cloneOptions, err := gitshared.GetRepoCloneOptions(context.Background(), credential, client, repoUrL, "master")
```

Comment:
Can you add a test for git clone from tag as well?

Reply:
Yes added .